content/enum/PostFlag.md

```diff
@@ -0,0 +1,126 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/PostFlag
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public enum PostFlag
+```
+
+Defines different states for an [`Post`](../class/Post.md).
+
+## Values
+
+### `DELETED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post was recently deleted, but is still in Memory.
+
+### `VIEWED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post was seen by the user. Used to determine where to place the viewport of the timeline viewer.
+
+### `LIKED`
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+This Post was liked by the active user.
+
+| API   | Twitter 1   | Twitter 2 | Mastodon    |
+| ----- | ----------- | --------- | ----------- |
+| Field | `favorited` |           | `favorited` |
+| Type  | bool        |           | bool        |
+
+### `REPOSTED`
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+This Post was reposted by the active user.
+
+| API   | Twitter 1   | Twitter 2 | Mastodon    |
+| ----- | ----------- | --------- | ----------- |
+| Field | `retweeted` |           | `reblogged` |
+| Type  | bool        |           | bool        |
+
+### `REPLIED`
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+This Post has an reply from the active user.
+
+### `HIDDEN_ON_TIMELINES`
```

Comment:
Is this different to the various "hidden" states that we have at the moment?  https://github.com/IBBoard/cawbird/blob/master/src/CbTweet.h#L32-L47

Does Mastadon let people post messages that are on their timeline (i.e. if you view a profile) but not in the home timeline feeds of anyone who follows them?

Reply:
The internal hiding states are not included yet (have to put this onto my todo).

Mastodon allows to hide posts from the "public" timeline (among other options). But since we would probably not include the public timeline anyway, we might need this only as an option for posting.